### PR TITLE
improve Uniswap V3 support

### PR DESCRIPTION
**Pull request by kenkunz**
# Summary

This PR updates the `trading-entities` collection with additional fields to support Uniswap V3. See tradingstrategy-ai/oracle#157 for additional background.

### Add `quality_factors` field

See tradingstrategy-ai/oracle#156

@miohtama suggested adding a flag called `liquidity-consideration-not-applicable`. After further thought, I think it's preferable to use a string array field as this gives us more flexibility to introduce other quality factors in the future (without having to add a separate flag for each one).

For now, the only valid `quality_factors` value is `"liquidity"`. This value would be set on all trading pairs _except_ Uniswap V3 pairs. When search results are returned to `frontend`, if a search result record includes a value of `"liquidity"` in the `quality_factors`, the record will be displayed as "low quality" in the UI if the liquidity is below a threshold value (currently a local constant of `50,000` in `frontend`). Since Uniswap V3 pairs _won't_ include this value, these records will not be flagged as "low quality".

### Add `pool_swap_fee` field

See #10 

The original suggestion was to call this field `fee_parcent` or `fee_basis`. I think `pool_swap_fee` is more descriptive. I followed the convention of other percent fields in the search index (e.g., `price_change_24h`) – stored as a `float` and expressed as a decimal – so a fee of `0.05%` is stored as `0.0005`.

### Remove unused `icon_url` field

When the search index was first created, we envisioned including an icon URL for every trading entity. This data is not currently available from our backend, so it's best to drop this field for now. It's trivial to add it back in the future.

### CODE REVIEW

**Comment by kenkunz** [comment]
FYI… it's hard to follow the diff to `trading-entities.md` – may be easier to review the markdown preview here:
https://github.com/tradingstrategy-ai/search/blob/10-uniswap-v3/docs/trading-entities.md

**Comment by kenkunz** [comment]
@miohtama any feedback on this before I merge these changes?

**Review by hieuh25 — APPROVED** [review]
LGTM!
